pkg/controller/nodenetworkstateconfiguration/nodenetworkstateconfiguration_controller.go

```diff
@@ -145,3 +156,88 @@ func (r *ReconcileNodeNetworkStateConfiguration) Reconcile(request reconcile.Req
 
 	return reconcile.Result{}, nil
 }
+
+/*
+Workaround Policy:
+
+
+cat <<EOF | ./kubevirtci/cluster-up/kubectl.sh apply -f -
+apiVersion: nmstate.io/v1alpha1
+kind: NodeNetworkConfigurationPolicy
+metadata:
+  name: brext-eth0-policy
+spec:
+  desiredState:
+    interfaces:
+    - name: brext
+      type: linux-bridge
+      state: up
+      ipv4:
+        dhcp: true
+        enabled: true
+      ipv6:
+        dhcp: true
+        enabled: true
+      bridge:
+        options:
+          vlan-filtering: true
+          vlans:
+          - vlan-range-min: 1
+            vlan-range-max: 4094
+        port:
+        - name: eth0
+          vlans:
+          - vlan-range-min: 1
+            vlan-range-max: 4094
+EOF
+
+*/
+
+// TODO in order to maintain compatibility in future, trigger workaround if the requested configuration matches
+func detectBridgeWorkaround(desiredState nmstatev1alpha1.State) (string, string, string, string) {
```

Comment:
I don't understand how the detection works. I expected to see here "if name == mad-workaround-policy", to make sure it does not kick in for run-of-the-mill policies.

Reply:
The issue with that would be that we would eventually need to introduce upgrade path or maintain the script forever. Instead, this specific scenario is handled by the script. Once we properly support it via nmstate, we can drop the workaround and this `desiredState` will do exactly the same job.

Reply:
Ok

Reply:
BTW I have tried it on bare-metal KNI and it worked! Not as a part of kubernetes-nmstate though, just the script alone, deployed as a DaemonSet.